package.json

```diff
@@ -1,12 +1,11 @@
 {
-  "name": "Sia-UI",
-  "version": "0.4.1",
-  "main": "index.js",
+  "name": "Sia-UI-beta",
+  "version": "0.4.2",
+  "main": "app/index.js",
   "author": "NebulousLabs",
   "description": "A UI application for interfacing with Sia",
   "license": "MIT",
   "devDependencies": {
-    "electron-packager": "latest",
```

Comment:
electron packager was putting things together wrong, and doing absurd things like resources/app/app/dependencies/Sia